collab-folder/src/core/view.rs

```diff
@@ -64,6 +64,40 @@ impl ViewsMap {
     self.remove_cache_view(parent_id);
   }
 
+  pub fn move_child_out(&self, parent_id: &str, view_id: &str) {
+    self.container.with_transact_mut(|txn| {
+      self.move_child_out_with_txn(txn, parent_id, view_id);
+    })
+  }
+
+  pub fn move_child_in(&self, parent_id: &str, view_id: &str, prev_id: Option<String>) {
+    self.container.with_transact_mut(|txn| {
+      self.move_child_in_with_txn(txn, parent_id, view_id, prev_id);
+    })
+  }
```

Comment:
Same comment as the previous one. 

I think moving out and moving in should be paired. Is there something that I missed?

Reply:
Because workspace and view is splited.